### ISSUE TITLE
Failure to find dev.appsody:java-microprofile:pom:0.2.11

### ISSUE THREAD

**Issue by jaricsng** [kabanero-io/guide-collection-microprofile#13]
followed the document [try-it](https://kabanero.io/guides/collection-microprofile/#initializing-your-project) and then ```appsody run```

```
appsody init java-microprofile
appsody run
```

encountered the following error.

```
[Container] [INFO] Scanning for projects...
[Container] [ERROR] [ERROR] Some problems were encountered while processing the POMs:
[Container] [FATAL] Non-resolvable parent POM for dev.appsody.starter.java-microprofile:starter-app:1.0-SNAPSHOT: Failure to find dev.appsody:java-microprofile:pom:0.2.11 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 8, column 13
[Container]  @
[Container] [ERROR] The build could not read 1 project -> [Help 1]
[Container] [ERROR]
[Container] [ERROR]   The project dev.appsody.starter.java-microprofile:starter-app:1.0-SNAPSHOT (/project/user-app/pom.xml) has 1 error
[Container] [ERROR]     Non-resolvable parent POM for dev.appsody.starter.java-microprofile:starter-app:1.0-SNAPSHOT: Failure to find dev.appsody:java-microprofile:pom:0.2.11 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 8, column 13 -> [Help 2]
[Container] [ERROR]
[Container] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[Container] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[Container] [ERROR]
[Container] [ERROR] For more information about the errors and possible solutions, please read the following articles:
[Container] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[Container] [ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
[Container] [Error] FATAL error APPSODY_PREP command received an error.  The controller is exiting: exit status 1
[Error] Error in 'appsody run': exit status 1
```

**Comment by jaricsng** [comment]
I changed the following version from **0.2.11** to **0.2.14** and the issues was resolved.

    <parent>
        <groupId>dev.appsody</groupId>
        <artifactId>java-microprofile</artifactId>
        <version>0.2.14</version>
    </parent>

**Comment by Charlotte-Holt** [comment]
@kilnerm Is there a specific version for this that users must have that we should specify in the guide?

**Comment by kilnerm** [comment]
@Charlotte-Holt This has been fixed in Collections releases from 0.2.0. Can we update the try it guide to use the following Kabanero index in the getting started section:

```
https://github.com/kabanero-io/collections/releases/download/0.2.1/kabanero-index.yaml
```

**Comment by Charlotte-Holt** [comment]
@kilnerm @jaricsng This update is live at https://kabanero.io/guides/collection-microprofile/. Closing this issue.
